consent-ws/src/main/java/org/broadinstitute/consent/http/models/DataSet.java

```diff
@@ -22,14 +22,29 @@
     @JsonProperty
     private Boolean active;
 
+    @JsonProperty
+    private String consentName;
+
     @JsonProperty
     private Boolean needsApproval;
 
+    @JsonProperty
+    private Integer alias;
+
```

Comment:
what is the alias typically used for?

Reply:
we use alias to create a user-friendly dataset UID for each dataset

Reply:
so which has the value represented by DUOS_00000x
is the alias=x?
or is the datasetId = x?
or both?

Reply:
it's alias